event.py

```diff
@@ -0,0 +1,271 @@
+                        webhooks=await eventchannel.webhooks()
+                        for webhook in webhooks:
+                            if webhook.name == "CalicobotLog":
+                                await webhook.send(embed=embed)
+                                return
+
+    @commands.Cog.listener
+    async def on_member_ban(guild: discord.Guild, user: discord.User):
+        async for action in guild.audit_logs(action=discord.AuditLogAction.ban):
+            embed=discord.Embed(title=f"`{user.global_name}`がBANされました",color=discord.Color.red(),timestamp=datetime.datetime.now())
+            embed.add_field(name="担当者",value=action.user.mention)
+            embed.add_field(name="理由",value=action.reason)
+            embed.set_author(icon_url=action.user.avatar.url,name=action.user)
+            embed.set_footer(text=guild.name)
+
+            botChannel=bot.get_all_channels()
+            for channel in botChannel:
+                if channel.guild == guild:
+                    if isinstance(channel, discord.TextChannel):
+                        webhooks=await channel.webhooks()
+                        for webhook in webhooks:
+                            if webhook.name == "CalicobotLog":
+                                await webhook.send(embed=embed)
+                                return
+
+    @commands.Cog.listener
+    async def on_member_join(member: discord.Member):
+        guild=member.guild
+        embed=discord.Embed(title=f"`{member.global_name}`が参加しました",color=discord.Color.red(),timestamp=datetime.datetime.now())
+        embed.add_field(name="アカウント作成日",value=f"<t:{int(time.mktime(member.created_at.timetuple()))}:R>",inline=False)
+        embed.add_field(name="ID",value=f"```{member.id}```",inline=False)
+        embed.set_thumbnail(url=member.avatar.url)
+        embed.set_footer(text=member.guild.name)
+
+        botChannel=bot.get_all_channels()
+        for channel in botChannel:
+            if channel.guild == guild:
+                if isinstance(channel, discord.TextChannel):
+                    webhooks=await channel.webhooks()
+                    for webhook in webhooks:
+                        if webhook.name == "CalicobotLog":
+                            await webhook.send(embed=embed)
+                            return
+
+    @commands.Cog.listener
+    async def on_member_unban(guild: discord.Guild, user: discord.User):
+        async for action in guild.audit_logs(action=discord.AuditLogAction.unban):
+            embed=discord.Embed(title=f"`{user.global_name}`がunBANされました",color=discord.Color.red(),timestamp=datetime.datetime.now())
+            embed.add_field(name="担当者",value=action.user.mention)
+            embed.add_field(name="ID",value=f"```{user.id}```")
+            embed.set_author(icon_url=action.user.avatar.url,name=action.user)
+            embed.set_footer(text=guild.name)
+
+            botChannel=bot.get_all_channels()
+            for channel in botChannel:
```

Comment:
According to Pylint, variable names must follow the snake_case naming convention.
```suggestion
            bot_channel=bot.get_all_channels()
            for channel in bot_channel:
```